src/main/daml/Daml/Finance/Interface/Settlement/Batch.daml

```diff
@@ -30,7 +30,7 @@ data View = View
 interface Batch where
   viewtype V
 
-  settle : Update [ContractId Transferable.I]
+  settle : Update [Optional (ContractId Transferable.I)]
```

Comment:
I think we could skip the `None` and just return the `Optional.catOptionals` instead. It makes it easier for a user to understand what's being returned

Reply:
agree